### ISSUE TITLE
Cannot chain nullable() method on Currency field

### ISSUE THREAD

**Issue by nonovd** [laravel/nova-issues#2361]
- Laravel Version: 5.8.35
- Nova Version: 2.11.0
- PHP Version: 7.3.13
- Database Driver & Version: pgsql 11.6
- Operating System and Version: Ubuntu 16.04.6 LTS
- Browser type and version: Firefox 73.0

### Description:

When using nullable() on a Currency field, we get the following error on **Index and Details** pages.

```json
{
    "message": "The given value "" does not represent a valid number.",
    "exception": "Brick\\Math\\Exception\\NumberFormatException",
    "file": "/var/www/vendor/brick/math/src/BigNumber.php",
    "line": 75
}
```
### Steps To Reproduce:

```php
    public function fields(Request $request)
    {
        return [
            Currency::make('Prix HT', 'amount_excl_tax')
                ->nullable()
                ->nullValues(["", "null", null])
                ->rules('required_without:amount_incl_tax', 'numeric', 'nullable')
                ->currency('EUR')
            ,
            Currency::make('Prix TTC', 'amount_incl_tax')
                ->nullable()
                ->nullValues(["", "null", null])
                ->rules('required_without:amount_excl_tax', 'numeric', 'nullable')
                ->currency('EUR')
            ,
        ];
    }
```

### Expected behavior

Should display a dash for null values.

**Comment by calvinmuller** [comment]
closed/fixed?

**Comment by jbrooksuk** [comment]
Fixed in v2.11.1
